schemas/group.yml

```diff
@@ -47,6 +47,10 @@
         beta: Reusing the `group` fields in this location is currently considered beta.
         normalize:
           - array
+      - at: process
+        as: attested_group
+        short_override: The attested group (agid).
```

Comment:
I guess agid is a new abbreviation? do we need to specify this, as it's not really a linux convention?

Reply:
I didn't think all of those were a linux convention. heh TIL. We could also not abbreviate it, I suppose.

Reply:
yea, see **man 7 credentials**

though looking i'm not really seeing a ruid (real user id), as I think that might just be uid, so we might have invented that, not sure. cc @m-sample

Reply:
I would avoid any suggestion that this related a Linux uids or gids at all;  Group names are all we get from k8s IIRC, not ids.

Should this be "attested_groups" - it is an array